cloudify_vsphere/devices/__init__.py

```diff
@@ -564,3 +564,100 @@ def detach_pci_device(ctx, **kwargs):
         else:
             raise NonRecoverableError(
                 'PCI Device can\'t be detached while VM is running')
+
+
+def _get_device_keys(vm, device_type):
+    device_keys = []
+    for device in vm.config.hardware.device:
+        if isinstance(device, device_type):
+            # if 'Hard disk 1' in device.deviceInfo.label:
```

Comment:
Why commented?

Reply:
will remove, that comment , it is not needed , the devices would be sorted correctly when listed from the vm